src/main/java/com/example/MainTask3.java

```diff
@@ -0,0 +1,18 @@
+//3.Вывести заданное количество случайных чисел с переходом и без перехода на новую строку.
+
+package com.example;
+
+import java.util.Scanner;
+
+public class MainTask3 {
+    public static void main(String[] args){
```

Comment:
Please add output of random number in one line, separated by whitespace (before or after output of numbers in multiple lines).
And let the numbers in one line and numbers in different lines be the same numbers in the same order.